data/data2.s

```diff
@@ -7177,14 +7178,78 @@ gUnknown_08376500:: @ 8376500
 	.incbin "baserom.gba", 0x00376500, 0x4
 
 gUnknown_08376504:: @ 8376504
-	.incbin "baserom.gba", 0x00376504, 0xd8
+	@ this is called as a list of items, but it has a few items that are not indexed until FRLG/Emerald.
```

Comment:
that doesn't make sense. I think these are move IDs, not item IDs. also, these should use constants, not integer literals